src/reference/config/writing-platform-configs.rst

```diff
@@ -60,114 +60,195 @@ What Are Install Targets?
-target.
+Note, if missing from configuration, the install target will default to the
+platform name. If incorrectly configured, this will cause errors in
+:ref:`RemoteInit`.
 
-But you might also have mirrored clusters, each with their own file system.
-Each cluster would be a platform with its own install target.
+To check if two platforms share a file system:
 
+Create a test file on the first platform:
 
-Example Platforms
------------------
+   .. code-block:: console
 
-On the Scheduler Host (Cylc Server)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+      $ ssh platform-A
+      $ echo "test file" > cylc-test-file
+      $ exit
 
-- **There is a built in localhost platform**
+Check the second platform for the file:
 
-.. admonition:: Scenario
+   .. code-block:: console
 
-   You want to allow users to submit small jobs to the scheduler host:
+      $ ssh platform-B
+      $ cat cylc-test-file
+      $ exit
 
-If a job doesn't set a platform it will run on the Cylc scheduler host
-using a default ``localhost`` platform.
+If the test file exists, then these two platforms share a file system and will
+require the same install target in ``global.cylc`` config file.
 
-Simple Remote Platform
-^^^^^^^^^^^^^^^^^^^^^^
+Example Platform Configurations
+-------------------------------
 
-- **Platforms don't need to be complicated**
-- ``install target`` **specifies a file system for the task using that platform**
+Detailed below are some examples of common platform configurations.
+
+Multiple Platforms Sharing File System with Cylc Scheduler
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **The Scheduler Host (Cylc Server) provides a built in localhost platform**
+- **Platform names can be defined as regular expressions.**
 
 .. admonition:: Scenario
 
-   Users want to run background jobs on a single server,
-   which doesn't share a file system with the workflow host.
+   Everyone in your organization has a computer called ``desktopNNN``,
+   all with a file system shared with the scheduler host. Many users
+   will want their desktop set up as a platform to run small jobs.
+
+In this scenario, Cylc does not need to install files on the desktop, since
+required files which are on the scheduler host will be accessible on the
+desktop. From Cylc's point of view, the desktop and scheduler hosts are
+considered different platforms but must share an :term:`install target`.
+Cylc needs to be told that these platforms share an install target and so we
+configure this using the designated configuration item:
+:cylc:conf:`global.cylc[platforms][<platform name>]install target`.
 
 .. code-block:: cylc
-   :caption: part of a ``global.cylc`` config file
+   :caption: the ``global.cylc`` config file for this scenario could look like:
 
    [platforms]
-       [[myhost]]
-           hosts = myhost
-           install target = myhost
+       [[localhost]]
+           hosts = localhost
+           install target = localhost
```

Comment:
These lines are not needed and should be removed.

Reply:
Ah, I see that they get removed in the text below.
I think it might be simpler to just explain that the localhost  platform does not need to be configured in this case since it is setup by default.